projects/sport-data-valley/seeds/protocols/rheumatism_one_time.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+questionnaire_keys = %w[intake_questionnaire_rheumatism ases_rheumatism braf_rheumatism eq5d5l_rheumatism hads_rheumatism]
+
+pr_name = File.basename(__FILE__)[0...-3]
+protocol = Protocol.find_by(name: pr_name)
+protocol ||= Protocol.new(name: pr_name)
+
+protocol.duration = 1.week
+protocol.informed_consent_questionnaire = nil
+protocol.invitation_text = 'Je bent uitgenodigd om een vragenlijst in te vullen.'
+protocol.save!
+
+bp_name = 'base-platform-subscription-rheumatism-one-time'
+bp_push_subscription = protocol.push_subscriptions.find_by(name: bp_name)
+bp_push_subscription ||= protocol.push_subscriptions.build(name: bp_name)
+bp_push_subscription.method = 'POST'
+bp_push_subscription.url = ENV['PUSH_SUBSCRIPTION_URL']
+bp_push_subscription.save!
+
+protocol.save!
+
+unused_measurement_ids = protocol.measurements.pluck(:id).to_set
+questionnaire_keys.each_with_index do |questionnaire_key, idx|
```

Comment:
based on the ticket, I understand that there should be a protocol per questionnaire. Are you sure this is how they want it?

Reply:
I asked Dennis about this if he wanted one protocol per questionnaire or a single one for all of them. He said that for now, 1 protocol is good. Later they would want to pull them apart because a few are probably suited for a wider audience.